legacy/src/main/java/org/opensearch/sql/legacy/plugin/RestSqlAction.java

```diff
@@ -234,7 +260,7 @@ private static boolean isExplainRequest(final RestRequest request) {
         return request.path().endsWith("/_explain");
     }
 
-    private static boolean isClientError(Exception e) {
+    public static boolean isClientError(Exception e) {
```

Comment:
why change to public?

Reply:
I updated this function to public to allow `RestSqlQueryAction` to output the same error format as this class. As well it seemed like a function which seemed appropriately public.